t/06-blacklist.t

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/perl
 
+use LedgerSMB::Sysconfig;
```

Comment:
I'm not sure I like this one. Can't you use File::Temp ? This test really doesn't have anything to do with the configuration of LedgerSMB...